appserver/ejb/ejb-http-remoting/client/pom.xml

```diff
@@ -86,5 +86,16 @@
             <groupId>javax.json</groupId>
             <artifactId>javax.json-api</artifactId>
         </dependency>
+
+       <dependency>
+           <groupId>junit</groupId>
+           <artifactId>junit</artifactId>
+       </dependency>
```

Comment:
Scope test?

Reply:
Right, thanks. I thought the scope is managed as well.